Formula/tezos-accuser-014-PtKathma.rb

```diff
@@ -9,9 +9,9 @@ class << self
   end
   homepage "https://gitlab.com/tezos/tezos"
 
```

Comment:
As we discussed, we want to follow the new upstream approach to not include the protocol number inside packages and I think that this should include brew formulae.

So, I believe that and existing formulae (like this one) need to be renamed and new ones (for lima) shouldn't have one already